cmd/vaultify/vaultify.go

```diff
@@ -148,6 +148,8 @@ func init() {
 	templateCmd.Flags().StringVar(&flags.templateOptions.Role, "role", "", "Vault kubernetes role to assume")
 	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateFileName, "template-file", "", "Template file to render")
 	templateCmd.Flags().StringVar(&flags.templateOptions.OutputFileName, "output-file", "", "Output file")
+	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateDirName, "template-dir", "", "Template diretory to render all files in")
```

Comment:
```suggestion
	templateCmd.Flags().StringVar(&flags.templateOptions.TemplateDirName, "template-path", "", "Template diretory to render all files in")
```

How about we have a generic flag. And based on whether the path is a file or directory we choose what to do?

In that case we could mark the old flags as deprecated :)

Reply:
I had the same thought, but was afraid of deprecating. But keeping the old and just mark them deprecated could work. I'll do that!